File: prak 7/c/PRAK605-2210817310005-IkhsanGymnastiar.c
#include<stdio.h>
int main(){
        int matriksA[10][10], matriksB[10][10], matriksAB[10][10];
        int a, b, c, d, jumlah=0;
        scanf("%d", &d);
        printf("Matriks A\n");
        for(a=0; a<d; a++){
          for(b=0; b<d; b++){
              scanf("%d", &matriksA[a][b]);}}  
        printf("Matriks B\n");
        for(a=0; a<d; a++){
          for(b=0; b<d; b++){
              scanf("%d", &matriksB[a][b]);}}
        for(a=0; a<d; a++){
          for(b=0; b<d; b++){
              for(c=0; c<d; c++){
                  jumlah = jumlah + matriksA[a][c] * matriksB[c][b];}      
                matriksAB[a][b]= jumlah;
                jumlah = 0;}}
        printf("Matriks AXB\n");
        for (a=0; a<d; a++){
          for(b=0; b<d; b++){
                    printf("%d ", matriksAB[a][b]);}
          printf("\n");}}
          

File: prak 7/python/PRAK605-2210817310005-IkhsanGymnastiar.py
n = int(input())
matriksA = []
print("Matriks A")
for i in range (n):
    k = list(map(int, input().split()))
    matriksA.append(k)
matriksB = []
print("Matriks B")
for i in range (n):
    k = list(map(int, input().split()))
    matriksB.append(k)
matriksAB = []
print("Matriks AXB")
for i in range (n):
    matriksAB.append([])
    for j in range (n):
        x = 0
        for k in range (n):
            x = x + matriksA[i][k] * matriksB[k][j]
        matriksAB[i].append(x)
for baris in matriksAB:
    for k in baris:
        print(k, end=' ')
    print()